### ISSUE TITLE
Texture Removal Failing

### ISSUE THREAD

**Issue by msj121** [aframevr/aframe#2652]
http://stackoverflow.com/questions/43774660/aframe-remove-texture

The issue is that removing a texture either doesn't work, or creates a "black" surface. You can see the code pen: https://codepen.io/msj121/pen/JNrWxg

```
//Neither work:
sphere.setAttribute("src","");
sphere.setAttribute('material', {src: ''});
```

Uses: https://aframe.io/releases/0.5.0/aframe.min.js

**Comment by nwoltman** [comment]
These don't work either:

```js
entity.removeAttribute('src');
entity.removeAttribute('material', 'src');
```

---

Also, if I have an entity that looks like this:

```jsx
<a-sky shader="flat" src="/image.jpg"></a-sky>
```

This works:

```js
entity.setAttribute('material', {}, true); // `shader` and `src` go back to their defaults
```

But this doesn't work:


```js
entity.setAttribute('material', {shader: 'flat'}, true);
```

So it seems like `src` doesn't get removed if the `shader` property doesn't change.

**Comment by nwoltman** [comment]
Looks like this is fixed in v0.6.x

**Comment by dmarcos** [comment]
Nice. Thank you
